froide/account/forms.py

```diff
@@ -214,12 +214,15 @@ def __init__(self, *args, **kwargs) -> None:
         )
 
 
+Replacements_1 = Dict[str, Union[str, bool]]
```

Comment:
Ah, sorry! `Replacements` in `text_utils` was an actual mapping of string replacements there, like `in str.replace`. Here `clean` returns cleaned form data, so you can name it e.g. `FormData`.